app/Models/Lesson.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Lesson extends Model
+{
+    use HasFactory, SoftDeletes;
+
+    protected $table = 'lessons';
+    protected $fillable = ['name'];
+
+    public function teachers()
+    {
+        return $this->hasOne(Teacher::class, 'teacher_id');
```

Comment:
có cần teacher_id ?

Reply:
return $this->hasMany(modelName::class, 'foreign_key', 'local_key');
Em chỉ viết đúng cấu trúc - Kb có sai k ạ?